guides/writing-addons/intro-tutorial.md

```diff
@@ -57,7 +57,7 @@ There are several options to see the addon in action. We could use `npm link` or
 **From the directory of the app using the addon:**
 1. `yarn link <addon-name>` or `npm link <addon-name>`.
 2. In the Ember app's `package.json`, add a `devDependencies` entry for your addon, like `"addon-name": "*"`. The `*` means that it will include all version numbers of our addon.
-3. Run `yarn install` or `npm install` in the app (for first time use `npm install --prefer-offline` or else `npm install --offline` instead of `npm install`. This is because `npm install` by default, will check online npm registry for your addon instead of your local storage)
+3. Run `yarn install` or `npm install` in the app (if using the app for the first time, use `npm install --prefer-offline` or `npm install --offline` instead of `npm install`. This is because `npm install` by default will check the online npm registry for your addon instead of your local storage)
```

Comment:
**suggestion**

I think the following changes could help improve the paragraph even more. What do you think?

```suggestion
3. Run `yarn install` or `npm install` in the app. (If you are using the app for the first time, you can use `npm install --prefer-offline` or `npm install --offline` instead. These alternative commands can speed up installation, because `npm install` checks the online npm registry for your addon instead of your local storage.)
```